app/assets/stylesheets/profile.scss

```diff
@@ -113,19 +115,23 @@
   }
 
   #profile {
-    padding: 20px;
     margin-bottom: 35px;
+    border-radius: 20px 20px 0px 0px;
 
     #profile_photo {
       margin-top: 10px;
-      padding-bottom: 10px;
       text-align: center;
     }
 
+    #profile_photo img {
```

Comment:
- Avoid using id selectors
- Merge rule `#profile_photo img` with rule on line 121
- Selector should have depth of applicability no greater than 3, but was 4
- Selector `profile_photo` should be written in lowercase with hyphens